server/internal/adapter/gql/resolver_mutation_project.go

```diff
@@ -112,33 +120,104 @@ func (r *mutationResolver) DeleteProject(ctx context.Context, input gqlmodel.Del
 
 func (r *mutationResolver) ExportProject(ctx context.Context, input gqlmodel.ExportProjectInput) (*gqlmodel.ExportProjectPayload, error) {
 
+	// create zip file instance
+	t := time.Now().UTC()
+	entropy := ulid.Monotonic(rand.New(rand.NewSource(uint64(t.UnixNano()))), 0)
+	name := ulid.MustNew(ulid.Timestamp(t), entropy)
+	zipFile, err := os.Create(fmt.Sprintf("%s.zip", name.String())) // reearth
```

Comment:
Please use https://github.com/spf13/afero instead of OS Filesystem thoughtout this implementation.